internal/api/api.go

```diff
@@ -5,12 +5,11 @@ package api
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/snyk/go-application-framework/internal/api/contract"
+	"github.com/snyk/go-application-framework/internal/constants"
 	"io"
 	"net/http"
 	"net/url"
-
-	"github.com/snyk/go-application-framework/internal/api/contract"
-	"github.com/snyk/go-application-framework/internal/constants"
```

Comment:
Let's keep these as they were please. Linter should catch this, but generally imports should be grouped by empty lines:

* stdlib packages
* 3rd-party packages
* local (in-module) packages